### PR TITLE
Added docker workflow for api

### PR DESCRIPTION
**Pull request by palbock**

### CODE REVIEW

File: .github/workflows/docker-push-dev.yml [hunk 1]
```diff
@@ -6,7 +6,9 @@ on:
       - master
 
 env:
-  IMAGE_NAME: alvnoas/alvtime-vue-pwa-dev
+  IMAGE_NAME_VUE: alvnoas/alvtime-vue-pwa-dev
+  IMAGE_NAME_API: alvnoas/alvtime-web-api-dev
```
Comment (TheKnarf) [inline .github/workflows/docker-push-dev.yml]: Maybe we should split this up into one pipeline per package?

File: Readme.md [hunk 2]
```diff
@@ -1,17 +1,15 @@
...
-[Frontend](./packages/frontend) | Frontend | Enduser frontend for Alvtime
-[Slack-app](./packages/slack-app) | Slack-app | Slack app
-[Api](./packages/api) | Backend | Backend api
-
+| Project                             | Type      | Description                  |
+| ----------------------------------- | --------- | ---------------------------- |
+| [Adminpanel](./packages/adminpanel) | Frontend  | Admin panel for Alvtime      |
+| [Frontend](./packages/frontend)     | Frontend  | Enduser frontend for Alvtime |
+| [Slack-app](./packages/slack-app)   | Slack-app | Slack app                    |
+| [Api](./packages/api)               | Backend   | Backend api                  |
```
Comment (TheKnarf) [inline Readme.md:12]: Was there any point in reformating this? I think formating it like you have done will make it harder to maintain in the future.

**Review by TheKnarf — CHANGES_REQUESTED** [review]
I suggest that we split up the github workflow per package instead of adding a lot of stuff to the same pipeline. I don't think there's any practical limit on how many pipelines we can have.

@Trulsaa may have some opinion if we should continue to build and push docker images from the github workflow or if it should be moved over to Azure.

Maybe we three should have a meeting to discuss this?

**Comment by palbock** [comment]
> 
> 
> I suggest that we split up the github workflow per package instead of adding a lot of stuff to the same pipeline. I don't think there's any practical limit on how many pipelines we can have.
> 
> @Trulsaa may have some opinion if we should continue to build and push docker images from the github workflow or if it should be moved over to Azure.
> 
> Maybe we three should have a meeting to discuss this?

Vi kan godt ha et møte. Per nå bygges docker images for backend i Azure og jeg ser noen fordeler med å flytte det over til github workflows.

File: Readme.md (hunk 2 above)
Comment (palbock) [reply Readme.md:12]: Det var ikke meningen å reformatere :S Det må VS code ha gjort av seg selv. Jeg rettet bare opp et par skriveleifer.

**Comment by Trulsaa** [comment]
Tar gjerne en prat om dette. Digger at du kjører på med workflows @palbock90. Jeg har også sett litt på muligheten for å dytte docker imaget direkte til app servicen. Men da bør vi nok gjøre det i devOps. Spent på hva dere tenker.

**Review by TheKnarf — APPROVED** [review]

**Pull request merged**